### DISCUSSION TITLE
Ver 6.0

### DISCUSSION THREAD

**Discussion by Stenodin** [flybywiresim/aircraft#4521] [Help] [answered]
I have downloaded Ver 6.0 via the installer but none of the new features are available?

Anyone have this issue?

**Comment by MeinTollerUsername** [comment, marked as answer]
Hey @Stenodin 

the A32nx is Not shown as a extra Aircraft in the MFSF Menu. Did you see that? You have to select the Flybywire A32nx instead of the Asobo A320. 

I hope that is helpfull

**Reply by robinottawa** [reply]
Not an extra **livery** of the Asobo. Now a separate aircraft.

**Reply by MeinTollerUsername** [reply]
Yes, i had a typo. I mean "now" instead of "Not" ;-)

**Comment by Stenodin** [comment]
Thanks I found it now. Wouldn’t have without your help.

On Wed, 21 Apr 2021 at 17:18, MeinTollerUsername ***@***.***>
wrote:

> Hey @Stenodin <https://github.com/Stenodin>
>
> the A32nx is no shown as a extra Aircraft in the MFSF Menu. Did you see
> that? You have to select the Flybywire A32nx instead of the Asobo A320.
>
> I hope that is helpfull
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/flybywiresim/a32nx/discussions/4521#discussioncomment-640898>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATF7GN3MK54TTU46Q73QOSTTJ33D7ANCNFSM43KLT5DA>
> .
>
-- 
Kevin T Williams

**Reply by MeinTollerUsername** [reply]
You are welcome. No problem 👍🏼
